src/app/util/attribute-storage.cpp

```diff
@@ -1275,9 +1287,9 @@ void emAfLoadAttributeDefaults(EndpointId endpoint, bool ignoreStorage, Optional
                         // uint16_t.  On big-endian platforms, a pointer to the default value of
```

Comment:
These comments need to be updated to reflect not only the storage in a uint32_t now, but the fact that this fix-up now applies to 2-byte attributes as well.